### PR TITLE
feat: add more command and task processing support

### PR DESCRIPTION
**Pull request by kyteinsky**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -19,20 +19,18 @@ This ensures that users can quickly grasp the essence of discussions without rea
...
 	<dependencies>
-		<nextcloud min-version="30" max-version="30"/>
+		<nextcloud min-version="30" max-version="31"/>
 	</dependencies>
 	<external-app>
 		<docker-install>
 			<registry>ghcr.io</registry>
-			<image>cloud-py-api/summarai</image>
-			<image-tag>1.0.0</image-tag>
+			<image>nextcloud/summarai</image>
```
Comment (marcelklehr) [inline appinfo/info.xml:27]: Should this be `nextcloud/sumupbot`?

File: Makefile [hunk 2]
```diff
@@ -10,51 +12,27 @@ help:
...
 	@echo "  "
 	@echo "  register          perform registration of running 'SummarAI' into the 'manual_install' deploy daemon."
-	@echo "  register_local    perform registration of running 'SummarAI' into the 'manual_install' deploy daemon for a local running instance."
-	@echo "  register27        perform registration of running 'SummarAI' into the 'manual_install' deploy daemon."
 
 .PHONY: build-push
 build-push:
 	docker login ghcr.io
-	docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ghcr.io/cloud-py-api/summarai:1.0.0 .
+	docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag ghcr.io/cloud-py-api/summarai:$(APP_VERSION) --tag ghcr.io/cloud-py-api/summarai:latest .
```
Comment (marcelklehr) [inline Makefile:24]: Should the URL be `nextcloud/sumupbot` here?

File: appinfo/info.xml (hunk 1 above)
Comment (kyteinsky) [reply appinfo/info.xml:27]: nice catch!

**Comment by marcelklehr** [comment]
Do we keep the scheduling mechanism? It feels very convoluted and bloats up the app, IMHO

**Comment by kyteinsky** [comment]
> Do we keep the scheduling mechanism? It feels very convoluted and bloats up the app, IMHO

It was already done and worked so I kept it for later consideration. It is bloated but is a nice automation feature.

**Review by marcelklehr — APPROVED** [review]
Did not test, but looks good

**Pull request merged**